packages/commonwealth/client/scripts/views/components/ProposalCard/helpers.tsx

```diff
@@ -131,5 +131,19 @@ export const getStatusText = (proposal: AnyProposal, isLoading?: boolean) => {
   return '';
 };
 
+export const shortenIdentifier = (identifer: string) => {
```

Comment:
You can add this function to packages/commonwealth/client/scripts/helpers/index.tsx if you want